src/camera-controls.js

```diff
@@ -804,6 +813,17 @@ export default class CameraControls extends EventDispatcher {
 		this.object.position.setFromSpherical( this._spherical ).add( this._target );
 		this.object.lookAt( this._target );
 
+		if ( this.boundaryEnclosesCamera ) {
+
+			this._encloseToBoundary(
```

Comment:
I think just `this.boundary.clampPoint` should be enough at here
```diff
-			this._encloseToBoundary(
-				this.object.position.copy( this._target ),
-				_v3A.setFromSpherical( this._spherical ),
-				1.0
-			);
+			this.boundary.clampPoint(
+				this.object.position,
+				this.object.position
+			);
```

Reply:
I prefer the current one. I don't like how the camera moves on the edge when its "friction" is zero

...also I kinda want to change the term "friction" ? How we should call him

Reply:
Sure keep it as is then👍